service/templates/admin.html

```diff
@@ -458,6 +458,35 @@ <h2 class="modal-title">[[modalTitle]]</h2>
 									</table>
 
 								</md-card-content>
+								<md-card-title>
+									<md-card-title-text>
+										<span>Disabled GQL Queries</span>
+									</md-card-title-text>
+								</md-card-title>
+								<md-card-content>
+									<div>
+										<section style="height: 70px;" layout="row" flex>
+											<md-input-container class="control"  style="width: 200px;">
+												<label>Operation name to disable</label>
```

Comment:
To clarify is this the resolver name? Will pages fail to load if we've disabled operations? Do we intend to remove this code once mainline is baked in or do we want to keep this as a degradable option?